infra/feast-operator/api/v1alpha1/featurestore_types.go

```diff
@@ -205,6 +209,15 @@ type OptionalConfigs struct {
 	Resources       *corev1.ResourceRequirements `json:"resources,omitempty"`
 }
 
+// AuthConfig defines the authorization settings for the deployed Feast services.
+type AuthConfig struct {
+	KubernetesAuth *KubernetesAuth `json:"kubernetes,omitempty"`
+}
+
+type KubernetesAuth struct {
+	Roles []string `json:"roles,omitempty"`
+}
```

Comment:
This needs a description via comment. `Roles` could use a description as well, along with details on any default we might set (if any). Maybe even explaining to the user that these correlate into deployed `Role` objects?